src/data/faq.json

```
@@ -1403,16 +1403,16 @@
                         "anchor": "why_oss",
                         "active": true,
                         "textblock": [
-                            "We believe that the success of the app depends directly on user acceptance and trust. With an open source approach, the full source code for the app and the infrastructure are freely available without access restrictions. In this way, we create the transparency necessary for a strong basis of trust.",
-                            "Furthermore, the open-source approach gives the broader public and the developer community the opportunity to actively contribute to the app’s success, for example, through reviews and pull requests."
+                            "We believe that the success of the app depends directly on user acceptance and trust. With an open source approach, the full source code for the app and the infrastructure are freely and openly available without access restrictions. In this way, we create the transparency necessary for a strong basis of trust.",
+                            "Furthermore, the open-source approach gives the broader public and the developer community the opportunity to actively contribute to the app’s success, for example, through reporting of errors, discussions, code reviews and contributions via pull requests."
                         ]
                     },
                     {
                         "title": "Can I really see all source code of the project?",
                         "anchor": "oss_complete",
                         "active": true,
                         "textblock": [
-                            "Yes. All code central to risk identification and propagation functionality is available openly on GitHub. Reviews are possible before and after the app is released. Questions about topics such as security, data privacy will be clarified with the community. Each person can create their own build of the app and test it. Apart from the source code, we also discuss central issues there. The discussion posts remain publicly available on the GitHub."
+                            "Yes. The full source code of the app is openly available on <a href='https://github.com/corona-warn-app'>GitHub</a>. Code reviews and comments are possible before and after a version of the app is released. Questions about development topics such as security, data privacy will be clarified with the community. Each person can create their own build of the app and test it. Apart from the source code, we also discuss other central issues relevant for the app there. The discussion posts remain publicly available on the GitHub."

```

I would remove "the" before GitHub (in the last line) to be consistent with "is openly available on GitHub" (first line). 

```suggestion
                            "Yes. The full source code of the app is openly available on <a href='https://github.com/corona-warn-app'>GitHub</a>. Code reviews and comments are possible before and after a version of the app is released. Questions about development topics such as security, data privacy will be clarified with the community. Each person can create their own build of the app and test it. Apart from the source code, we also discuss other central issues relevant for the app there. The discussion posts remain publicly available on GitHub."
```